backend/views/core/invoices/create.py

```diff
@@ -55,10 +55,7 @@ def invoice_page_post(request: HttpRequest):
         invoice.client_city = request.POST.get("to_city")
         invoice.client_county = request.POST.get("to_county")
         invoice.client_country = request.POST.get("to_country")
-        if request.POST.get("is_representative") == "on":
-            invoice.client_is_representative = True
-        else:
-            invoice.client_is_representative = False
+        invoice.client_is_representative = True if request.POST.get("is_representative") == "on" else Fals
```

Comment:
Can this bug be fixed please - `False` instead of `Fals`

Reply:
Yes, of course, I just did noticed that and will fix it right away

Reply:
Typo fixed and now all checks passed